test/paintings.test.js

```diff
@@ -21,3 +21,44 @@ describe('Get all Paintings', () => {
       });
   });
 });
+
+describe('Get artist paints by id', () => {
+  test('Route /paintings/1 status 200, json header, data.name = ca-wiki ', (done) => {
```

Comment:
data.name = ca-wiki !! :wink:

Reply:
Oops, I didn't pay attention for this haha Lol, Thank you anyway *.*

Reply:
Please check if every thing is correct now, and what you mentioned before has been solved.